src/index.js

```diff
@@ -1,17 +1,61 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
-import App from './App';
-import * as serviceWorker from './serviceWorker';
+import Map from './components/Map'
+
+class Home extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
In a function component this would be something like `[location, setLocation] = useState({latitude: null, longitude: null});`